### ISSUE TITLE
Fan control has stopped working with kernel 5.15

### ISSUE THREAD

**Issue by aardbol** [tuxedocomputers/tuxedo-control-center#224]
It's running at a constant speed as a result

**Comment by tuxedoxt** [comment]
Hello,

maybe the driver package `tuxedo-keyboard` did not correctly build for the new kernel

Check that it's marked as installed by dkms for the currently booted kernel
```
dkms status
uname -r
```

If not, reinstall the driver package, for ubuntu `apt install tuxedo-keyboard --reinstall`.

**Comment by goto-loop** [comment]
Got the same problem here with kernel 5.19. I've already reinstalled both `tuxedo-keyboard` and `tuxedo-control-center` and dkms shows the driver as installed, but it's not working.

**Comment by tuxedoxt** [comment]
> Got the same problem here with kernel 5.19. I've already reinstalled both `tuxedo-keyboard` and `tuxedo-control-center` and dkms shows the driver as installed, but it's not working.
> 
> EDIT: Could this be a module loading issue? `dkms status` shows `tuxedo_keyboard`, but shouldn't there also be `tuxedo_io` for the fan control to work?

Yes, currently the modules `tuxedo_keyboard`, `tuxedo_io` and some interface module depending on device should be loaded.

The modules get explicitly loaded on package upgrade by the installer. On a normal boot they should be auto-loaded based on the assigned module aliases. So far I have not seen the module alias auto-load fail for relevant devices.

**Comment by un-kasimir** [comment]
@tuxedoxt can you show a sample output of `dkms status`?
For me, `tuxedo_io` isn't listed as well.
